.github/workflows/cicd.yml

```diff
@@ -23,5 +23,14 @@ jobs:
       - name: Install dependencies
         run: yarn --immutable
 
+      - name: Build packages
+        run: yarn build:packages
```

Comment:
Is `build:packages` required for the new steps we added below?

Reply:
Tests need it so that each package has the latest version of its dependencies (e.g. React has the latest build of React-core). If I understand correctly, turbo should automatically take care of that because in `turbo.json` we list build as a dependency of test, but I added here just to make it clear we're doing a build and that it gets listed as a separate step.